ANT+_Android_SDK/Sample/ANT+ Plugin Sampler/source/src/main/java/com/dsi/ant/antplus/pluginsampler/pulsezones/Activity_PulseZonesFitness.java

```diff
@@ -151,11 +156,23 @@ public void subscribeToHrEvents() {
             }
 
             runOnUiThread(() -> {
+                //Synchronization of starting time of the readings and chronometer start
+                if(!chronometer.isRunning()) {
+                    startTimeInMillisec = System.currentTimeMillis();
```

Comment:
I'm wondering if we can rely on only one timer ... for instance, if we can get the time from the chronometer instead ?

Reply:
no